LathBotFront/Commands/ReactionCommands.cs

```diff
@@ -95,7 +103,7 @@ public async Task Repeat(CommandContext ctx, [Description("What the bot should r
         [Description("Parth!")]
         public async Task Parth(CommandContext ctx)
         {
-            await ctx.Channel.SendMessageAsync("Boss man! (even tho the whole senate technically are all equal but i am keeping this for the laughs)");
+            await ctx.Channel.SendMessageAsync("Boss man! (even tho the whole senate technically are all equal but i am keeping this for the laughs, also he is the actual owner)");
```

Comment:
no need to state this really
```suggestion
            await ctx.Channel.SendMessageAsync("Boss man! (even tho the whole senate technically are all equal but i am keeping this for the laughs)");
```